### PR TITLE
FAT2-297 campaign outer api

### PR DESCRIPTION
**Pull request by dashton82**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -932,6 +1002,22 @@ stages:
   #     ProductId: $(RecruitProductId)
   #     ApplicationIdentifierUri: $(RecruitOuterApiIdentifierUri)
   
+  #  - template: pipeline-templates/job/deploy-api.yml
+  #    parameters:
+  #      DeploymentName: Deploy_Campaign
+  #      DeploymentCondition: and(succeeded(), eq(${{ parameters.Campaign }}, true))
+  #      DeploymentApprovals: PROD
+  #      ServiceConnection: SFA-DAS-Prod-ARM
```
Comment (adam230594) [inline azure-pipelines.yml:1010]: Needs to be SFA-DIG-Prod-ARM

File: azure-pipelines.yml [hunk 2]
```diff
@@ -1090,4 +1176,21 @@ stages:
...
-    #     ApplicationIdentifierUri: $(RecruitOuterApiIdentifierUri)
\ No newline at end of file
+    #     ApplicationIdentifierUri: $(RecruitOuterApiIdentifierUri)
+    
+    #  - template: pipeline-templates/job/deploy-api.yml
+    #    parameters:
+    #      DeploymentName: Deploy_Campaign
+    #      DeploymentCondition: and(succeeded(), eq(${{ parameters.Campaign }}, true))
+    #      DeploymentApprovals: MO
+    #      ServiceConnection: SFA-DAS-ModelOffice-ARM
```
Comment (adam230594) [inline azure-pipelines.yml:1186]: Needs to be SFA-ASM-ModelOffice-ARM. Could you also fix this on the MO recruit deployment as well?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -771,6 +825,22 @@ stages:
       ProductId: $(RecruitProductId)
       ApplicationIdentifierUri: $(RecruitOuterApiIdentifierUri)
 
+  - template: pipeline-templates/job/deploy-api.yml
+    parameters:
+      DeploymentName: Deploy_Campaign
+      DeploymentCondition: and(succeeded(), eq(${{ parameters.Campaign }}, true))
+      DeploymentApprovals: DEV
+      ServiceConnection: SFA-DAS-PreProd-ARM
```
Comment (adam230594) [inline azure-pipelines.yml:833]: Should be SFA-DIG-PreProd-ARM

File: azure-pipelines.yml (hunk 1 above)
Comment (dashton82) [reply azure-pipelines.yml]: 👍🏻

File: azure-pipelines.yml (hunk 3 above)
Comment (dashton82) [reply azure-pipelines.yml]: 👍🏻

File: azure-pipelines.yml (hunk 2 above)
Comment (dashton82) [reply azure-pipelines.yml]: 👍🏻

File: src/SFA.DAS.Campaign/InnerApi/Requests/GetSectorsListRequest.cs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.Campaign.InnerApi.Requests
+{
+    public class GetSectorsListRequest : IGetApiRequest
```
Comment (scottywakefield) [inline src/SFA.DAS.Campaign/InnerApi/Requests/GetSectorsListRequest.cs:5]: can be moved to shared

**Review by scottywakefield — APPROVED** [review]

**Pull request merged**